plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraTypeMapping.java

```diff
@@ -0,0 +1,596 @@
+    {
+        SqlDataTypeTest.create()
+                .addRoundTrip("set<text>", "NULL", VARCHAR, "CAST(NULL as varchar)")
+                .addRoundTrip("set<text>", "{}", VARCHAR, "CAST(NULL as varchar)")
+                .addRoundTrip("set<text>", "{'Trino'}", VARCHAR, "CAST('[\"Trino\"]' as varchar)")
+                .execute(getQueryRunner(), cassandraCreateAndInsert("tpch.test_set"));
+    }
+
+    @Test
+    public void testCassandraMap()
+    {
+        SqlDataTypeTest.create()
+                .addRoundTrip("map<text, text>", "NULL", VARCHAR, "CAST(NULL as varchar)")
+                .addRoundTrip("map<text, text>", "{}", VARCHAR, "CAST(NULL as varchar)")
+                .addRoundTrip("map<text, text>", "{'connector':'cassandra'}", VARCHAR, "CAST('{\"connector\":\"cassandra\"}' as varchar)")
+                .execute(getQueryRunner(), cassandraCreateAndInsert("tpch.test_map"));
+    }
+
+    @Test
+    public void testCassandraInet()
+    {
+        SqlDataTypeTest.create()
+                .addRoundTrip("inet", "NULL", createVarcharType(45), "CAST(NULL AS varchar(45))")
+                .addRoundTrip("inet", "'0.0.0.0'", createVarcharType(45), "CAST('0.0.0.0' AS varchar(45))")
+                .addRoundTrip("inet", "'116.253.40.133'", createVarcharType(45), "CAST('116.253.40.133' AS varchar(45))")
+                .addRoundTrip("inet", "'255.255.255.255'", createVarcharType(45), "CAST('255.255.255.255' AS varchar(45))")
+                .addRoundTrip("inet", "'::'", createVarcharType(45), "CAST('::' AS varchar(45))")
+                .addRoundTrip("inet", "'2001:44c8:129:2632:33:0:252:2'", createVarcharType(45), "CAST('2001:44c8:129:2632:33:0:252:2' AS varchar(45))")
+                .addRoundTrip("inet", "'ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff'", createVarcharType(45), "CAST('ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff' AS varchar(45))")
+                .addRoundTrip("inet", "'ffff:ffff:ffff:ffff:ffff:ffff:255.255.255.255'", createVarcharType(45), "CAST('ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff' AS varchar(45))")
+                .execute(getQueryRunner(), cassandraCreateAndInsert("tpch.test_inet"));
+    }
+
+    @Test
+    public void testCassandraVarint()
+    {
+        SqlDataTypeTest.create()
+                .addRoundTrip("varint", "NULL", VARCHAR, "CAST(NULL AS varchar)")
+                .addRoundTrip("varint", "12345678910", VARCHAR, "CAST('12345678910' AS varchar)")
+                .addRoundTrip("varint", "2147483648", VARCHAR, "CAST('2147483648' AS varchar)")                     // Integer.MAX_VALUE + 1
+                .addRoundTrip("varint", "-2147483649", VARCHAR, "CAST('-2147483649' AS varchar)")                   // Integer.MIN_VALUE - 1
+                .addRoundTrip("varint", "9223372036854775808", VARCHAR, "CAST('9223372036854775808' AS varchar)")   // Long.MAX_VALUE + 1
+                .addRoundTrip("varint", "-9223372036854775809", VARCHAR, "CAST('-9223372036854775809' AS varchar)") // Long.MIN_VALUE - 1
```

Comment:
nit : We generally don't format the comment in such style. 
```suggestion
                .addRoundTrip("varint", "2147483648", VARCHAR, "CAST('2147483648' AS varchar)") // Integer.MAX_VALUE + 1
                .addRoundTrip("varint", "-2147483649", VARCHAR, "CAST('-2147483649' AS varchar)") // Integer.MIN_VALUE - 1
                .addRoundTrip("varint", "9223372036854775808", VARCHAR, "CAST('9223372036854775808' AS varchar)") // Long.MAX_VALUE + 1
                .addRoundTrip("varint", "-9223372036854775809", VARCHAR, "CAST('-9223372036854775809' AS varchar)") // Long.MIN_VALUE - 1
```